integration_test/integration_test.go

```diff
@@ -204,7 +233,31 @@ checks:
 				},
 			},
 			{
-				Name: `legacy configuration with args other than "ignore" fails`,
+				Name: `legacy configuration with "exclude" arg is upgraded`,
+				ConfigFiles: map[string]string{
+					"godel/config/check.yml": `
+checks:
+  errcheck:
+    args:
+      - "-exclude"
+      - "testdata/errcheck.exclude"
```

Comment:
This is in `testdata` because the test runner doesn't change the working directory to `projectDir`, so we can't create this as a `ConfigFile`.